media/media/20190402-175447-4585677096.html

```diff
@@ -0,0 +1,119 @@
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    
+        <title>Please verify your email</title>
+    
+
+    <style type="text/css">
+        #container {
+            margin: 10px auto;
+        }
+
+        #message_headers {
+            background: #fff;
+            font-size: 12px;
+            font-family: "Segoe UI", "Helvetica Neue", Arial, sans-serif;
+            border-bottom: 1px solid #dedede;
+            overflow: auto;
+        }
+
+        .message_container {
+            margin: 5px auto;
+
+            min-width: 400px;
+        }
+
+        .row {
+            display: block;
+            overflow: auto;
+        }
+
+        .col-left, .col-right {
+            float: left;
+            min-height: 1px;
+            overflow: auto;
+        }
+
+        .col-left {
+            width: 100px;
+            text-align: right;
+        }
+
+        .col-right {
+            min-width: 200px;
+            padding-left: 10px;
+        }
+
+        pre#message_body {
+            padding: 4px;
+            white-space: pre-wrap;
+            border: 1px solid #eee;
+            background-color: #fcfcfc;
+        }
+    </style>
+</head>
+<body>
+<div id="container">
+    <div id="message_headers">
+        <div class="message_container">
+            <div class="row">
+                <div class="col-left">From:</div>
+                <div class="col-right">verify@authorsheaven.com</div>
+            </div>
+            <div class="row">
+                <div class="col-left">Subject:</div>
+                <div class="col-right"><b>Please verify your email</b></div>
+            </div>
+            <div class="row">
+                <div class="col-left">Date:</div>
+                <div class="col-right">Sep 2, 2014 10:35:58 PM WIB</div>
+            </div>
+            <div class="row">
+                <div class="col-left">To:</div>
+                <div class="col-right">[&#39;thecliffcastle@gmail.com&#39;]</div>
+            </div>
+            
+        </div>
+    </div>
+
+    
+
+    
+        
+            
+                
+                    Hi Sichangi
```

Comment:
Shouldn't this be the username the user has used to create the account?

Reply:
Also, kindly remove it from this PR

Reply:
> Also, kindly remove it from this PR

Okey